examples/julia_logo.jl

```diff
@@ -0,0 +1,159 @@
+using Javis
+using DelimitedFiles
+using OffsetArrays, FFTViews # (not strictly necessary, but make DFT indexing more intuitive)
+
+# Import the discrete fourier transform of our points and recast them in their original complex form
+ps = readdlm("julia_logo.csv", ',')
+cs = complex.(ps[:, 1], ps[:, 2]) |> FFTView
+
+vid = Video(800, 500)
+
+function ground(args...)
+    background("black")
+    sethue("black")
+    circle(Point(0, 0), 200, :fill)
```

Comment:
feels odd to draw a black circle on a black background or in general having black as the default color.

Reply:
No reason, was also left from old code. I just thought yellow on black background was nicer, but we can change that as well

Reply:
I like the yellow one black 👌